pkg/syncer/cnsoperator/controller/cnsregistervolume/cnsregistervolume_controller.go

```diff
@@ -505,6 +512,116 @@ func (r *ReconcileCnsRegisterVolume) Reconcile(ctx context.Context,
 					storageClassName, vc.Config.Host, err)
 				return reconcile.Result{RequeueAfter: timeout}, nil
 			}
+
+			cnsVolumeInfoCRList := &cnsvolumeinfov1alpha1.CNSVolumeInfoList{}
+			restConfig, err := clientConfig.GetConfig()
```

Comment:
Why do we need to list the CNVolumeInfo instances here?

Reply:
Removed some stale code